src/sass/components/_section-brands.scss

```diff
@@ -0,0 +1,29 @@
+.section-brands {
+  margin-bottom: 20px;
+  display: flex;
+  .boxbrands {
+    border-top: 3px solid #cccccc;
+    border-bottom: 3px solid #cccccc;
+    margin-bottom: 10px;
+    opacity: 0.9;
+    button {
+      height: 100px;
+      width: 40px;
+      background-color: #2a2a2a;
+      color: white;
+      opacity: 1;
+      &:hover {
+        color: #e0a38e;
+      }
+    }
+  }
+}
+
+.brandimg {
+  width: 150px;
+  height: 100px;
+  margin: 0px 6px 0px 10px;
+}
+.line {
+  margin: 20px 0px 20px 0px;
```

Comment:
możemy tutaj użyć składni skróconej:
```
margin: 20px 0;
```
jeśli mamy `0px` to omijamy `px` :)

Reply:
Poprawiłem :)